projects/ngx-grapesjs/schematics/ng-add/index.ts

```diff
@@ -13,18 +13,13 @@ export const ngAdd =
     const grapesJsAssetPaths = [`${baseGrapesjsPath}/grapes.min.js`];
 
     if (options.editorType === 'webpageEditor') {
-      grapesCssAssetPaths.push(
-        `${baseGrapesjsWebpagePath}/grapesjs-preset-webpage.min.css`
-      );
       grapesJsAssetPaths.push(
-        `${baseGrapesjsWebpagePath}/grapesjs-preset-webpage.min.js`
+        `${baseGrapesjsWebpagePath}/index.js`,
```

Comment:
Since the `baseGrapesjsWebpagePath` is used only once, we can set its value directly here. BTW, is there a minified version of `index.js`?